src/index.js

```diff
@@ -37,13 +22,14 @@ function createTask(task) {
 }
 
 // Call functions
-list.forEach((task) => createTask(task));
+Object.values(list.list).forEach((value) => createTask(value));
```

Comment:
- [ ] When rendering the list of tasks please check if there is an existing item in the local storage to persist the state of your application with the local storage (currently when I reload the page the tasks completed are not coordinated with the local storage), I'd suggest using the `window.onload` function.
![image](https://user-images.githubusercontent.com/33767951/144465683-5b5358db-2706-45c0-b42a-d81720dfbade.png)